This package had an absolute path for sys/types.h, which doesn't
make much sense. It breaks on newer Ubuntu systems, and probably many
others once multiarch becomes more common.

This patch makes the types a relative path, and allows the system
to use whatever include paths it feels are correct.

diff -Naurp elftosb-10.12.01-orig/common/stdafx.h elftosb-10.12.01/common/stdafx.h
--- elftosb-10.12.01-orig/common/stdafx.h	2012-07-12 13:30:10.990249396 -0400
+++ elftosb-10.12.01/common/stdafx.h	2012-07-12 13:30:06.858249391 -0400
@@ -27,7 +27,7 @@
 // For Linux systems only, types.h only defines the signed
 // integer types.  This is not professional code.
 // Update: They are defined in the header files in the more recent version of redhat enterprise gcc.
-#include "/usr/include/sys/types.h"
+#include <sys/types.h>
 #include <stdint.h>
 //typedef unsigned long uint32_t;
 //typedef unsigned short uint16_t;
